test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -223,6 +214,72 @@ TEST(help_add_test, add_option)
                            "SeqAn version: 3.0.0");
     EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
                                expected | ranges::view::remove_if(is_space)));
+}
+
+TEST(test_add_copyright, copyright)
+{
+    std::string stdout;
+    std::string expected;
+
+    // Tests the --copyright call.
+    const char * argvCopyright[] = {"./copyright", "--copyright"};
+    argument_parser copyright("copyright", 2, argvCopyright);
 
-   // EXPECT_THROW(parser6.parse(), parser_interruption);
+    std::ifstream license_file{std::string{{SEQAN_INCLUDE_DIR}} + "/../LICENSE"};
+    std::string license_string{(std::istreambuf_iterator<char>(license_file)),
+                                std::istreambuf_iterator<char>()};
+    // Test --copyright with empty short and long copyright info.
+    {
+        testing::internal::CaptureStdout();
+        EXPECT_THROW(copyright.parse(), parser_interruption);
+        stdout = testing::internal::GetCapturedStdout();
+        expected = std::string("================================================================================"
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright copyright information not available."
```

Comment:
there is a double copyright? If this is the app name please change the app name to something like `MyApp` so it is clearer and also I would propose a `:`
-> `MayApp: copyright information not available.`